src/css/components/_post-card.scss

```diff
@@ -0,0 +1,104 @@
+.post-card p {
+  margin: 0;
+}
+
+.post-card__image-wrapper {
+  overflow: hidden;
+  width: 100%;
```

Comment:
One thing to be mindful of is how many ways do we want to introduce for dimensions, vh, px, % and rem.  I realize it's a ball buster but just for consistencies sake moving forward what are your thoughts?

Reply:
Yeah that's something I've been trying to research and figure out myself as well, I think the rule that I will try to follow is this one: 
`px` → Border widths, shadows
`rem` → margin and padding, font-size, min/max sizes
`em` → specific cases like if we have an element that should scale depending on the size of the element it is in (e.g.: an inline button)
`%` → Width and height of elements with a fixed size parent (e.g.: images in cards)
`vh/vw` → For larger container layouts

It's not a set-in-stone rule, but probably applies for 80% of the cases, what do you think?